### DISCUSSION TITLE
Sizing problem

### DISCUSSION THREAD

**Discussion by xCzakix** [arkypita/LaserGRBL#2305] [Q&A]
Hello,
I recently purchased model JL7
My question is if there is any way of changing the size of engraving to bigger than 5cm x 5cm?

**Comment by Tribo-2dot0** [comment]
Seria WAINLUX JL7? Essa máquina não tem 40x40cm de área de trabalho?

**Reply by xCzakix** [reply]
Yes it has this sizing but when I’m putting size more than 5cm in the program then it just reset itself and goes by 5cm anyway

**Reply by Tribo-2dot0** [reply]
> Sim ele tem esse tamanho, mas quando estou colocando tamanho superior a 5cm no programa então ele só se reseta e passa por 5cm de qualquer maneira

Digite $$ no console e cole o resultado aqui.
